draft-ietf-jsonpath-base.md

```diff
@@ -208,8 +210,19 @@ Query:
 Argument:
 : Short name for the value a JSONPath expression is applied to.
 
+Location:
+: the position of a value within the argument. This can be thought of
+  as a sequence of names and indexes navigating to the value through
+  the objects and arrays in the argument, with the empty sequence
+  indicating the argument itself.
+  A location can be represented as a Normalized Path (defined below).
+
 Node:
 : The pair of a value along with its location within the argument.
+  Member names do not have locations and so do not have nodes.
+  Members are not values and so do not have nodes.
+  Note that a node is associated with either the argument or one of
```

Comment:
I'm not entirely happy with "is associated", but don't have a much better idea.
Maybe "relates to"?  "comprises"?

Reply:
I reworded this to avoid the awkward relationship.